src/fing.cpp

```diff
@@ -43,7 +43,7 @@ void Fing::Reset()
 
 bool Fing::IsSupportedChild(Object *child)
 {
-    if (!child->Is({ REND, TEXT })) {
+    if (child->Is({ LB, REND, TEXT })) {
```

Comment:
This change flips the logic and is wrong. Please fix it, or even better make the check consistent with other `::IsSupportedChild` methods

Reply:
Thanks for the review. 
(Reminder to myself: Quick copy paste always needs checking.)